src/RecordComponent.cpp

```diff
@@ -176,6 +178,39 @@ RecordComponent::empty() const
     return *m_isEmpty;
 }
 
+void
+RecordComponent::serialize()
+{
+    // verify: read-only
+    if( IOHandler()->m_frontendAccess != Access::READ_ONLY )
+        throw std::runtime_error("serialize only works on read-only access");
+    //if( IOHandler()->m_backendAccess != Access::READ_ONLY )
+    // as well??
+
+    // verify: no MPI
+    auto & series = this->retrieveSeries();
+    auto const str_backend = series.backend(); // HDF5, ADIOS2, ...
+    if( str_backend.find( "MPI" ) != std::string::npos )
+        throw std::runtime_error("serialize does not work with MPI");
+
+    // information to construct a new serial read-only series
+    auto const str_path = IOHandler()->directory; // samples/git-samples/
+    auto const str_name = series.name();          // data%T
+    auto const str_ext = suffix(series.m_format); // .bp, .h5, .json, ...
+    auto const filepath = str_path + str_name + str_ext;
+
+    // information to find iteration
+    //auto const int it = ...; // 400
```

Comment:
```suggestion
    std::vector< std::string > const my_path = this->myPath();
    int const  it = std::stoi(my_path.at(1));  // 400
    
    std::string str_r, str_rc, str_part;
    if( my_path.at(2) == "meshes" )
    {
        str_r = my_path.at(3);  // "E"
        str_rc = my_path.at(4);  // "x"
    } else
    {
        str_part = my_path.at(3);  // "x"
        str_r = my_path.at(4);  // "E"
        str_rc = my_path.at(5);  // "x"
    }
```

Probably a good strategy: serialize the whole `vector< string >` and de-clutter during re-initalization/de-serialization.